HopsanNG.pro

```diff
@@ -1,4 +1,4 @@
 TEMPLATE = subdirs
 
 CONFIG += ordered
-SUBDIRS = HopsanCore componentLibraries SymHop Ops HopsanGenerator hopsangeneratorgui hopsanremote HopsanGUI HopsanCLI HoLC UnitTests
+SUBDIRS = HopsanCore componentLibraries SymHop Ops HopsanGenerator hopsangeneratorgui hopsanremote HopsanGUI HopsanCLI HoLC UnitTests hopsanc
```

Comment:
I am not sure if it would work, but it would be nice if the C-API exists in the core library directly, to avoid a wrapper DLL.
But on the other hand, its strange if the HopsanCore library contains an instance of it self, so a wrapper DLL might be a good idea anyway.

Reply:
I prefer a separate wrapper. I think it is cleaner and easier to maintain.

I think we have discussed it before, but a future project could be to only have a C API for HopsanCore. That would require completely re-writing the interface between HopsanGUI and HopsanCore though.